Makefile

```diff
@@ -108,31 +109,28 @@ cert-mgr: ## Install the certification manager
 ##################
 # Build and Load #
 ##################
-.PHONY: build bin/plain bin/unpack bin/core build-local-container kind-load kind-load-bundles kind-cluster
+.PHONY: build plain unpack core build-container kind-load kind-load-bundles kind-cluster
 
 ##@ build/load:
 
 # Binary builds
-GO_BUILD := $(Q)go build
 VERSION_FLAGS=-ldflags "-X $(VERSION_PATH).GitCommit=$(GIT_COMMIT)"
-build: bin/plain bin/unpack bin/core
-
-bin/plain:
-	CGO_ENABLED=0 go build $(VERSION_FLAGS) -o $@$(BIN_SUFFIX) ./internal/provisioner/plain
+build: plain unpack core
 
-bin/unpack:
-	CGO_ENABLED=0 go build $(VERSION_FLAGS) -o $@$(BIN_SUFFIX) ./cmd/unpack/...
+plain:
+	mkdir -p $(BIN_DIR)
```

Comment:
should this command be a defined somewhere since we call it for each of the binary builds?

Reply:
I could add a `$(BIN_DIR)` target that does `mkdir -p $(BIN_DIR)`? I'm always a little gun-shy about adding more targets because I feel like that makes the Makefile harder to understand and mentally parse.

But I'm up for it if that's there's consensus to do that.

Reply:
Potentially dumb question: do we even need to create this bin/ directory? I was under the impression that go build would handle that for us implicitly.

Reply:
Good catch. Removed these because `go build` takes care of it.